src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,53 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  position: relative;
+  padding: $paddings;
+  border: 1px solid #000;
+  border-radius: 50%;
+
+  &__center {
+    position: absolute;
+    width: $sides / 15;
+    height: $sides / 15;
```

Comment:
square

Reply:
kek, actually it was a mix in, but at some point i changed it because files didn't want to import.
Ant the most funniest thing that after today lecture i know why they didn't worked :D